python/cli/prelude_cli/views/shared.py

```diff
@@ -1,12 +1,10 @@
-import sys
 import time
 import click
 import threading
 
-
+from rich.progress import Progress, SpinnerColumn, TextColumn
```

Comment:
small nit that you will find recorded nowhere. But David likes his imports sorted by length ha

Reply:
meant that this line should be below the functools one!